t/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
     command: dgraph alpha -o 100 --my=alpha1:7180 --zero=zero1:5180 --logtostderr
       -v=2 --raft "idx=1; group=1" --security whitelist=0.0.0.0/0 --acl secret-file=/data/acl_secret;access-ttl=3s
```

Comment:
`"secret-file=/data/acl_secret;access-ttl=3s"` has to be in quotes like I have put it

Reply:
good catch...  I saw a similar issue with audit logging.  we should make a note of this in the docs